.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+
+Fixes #  
+
+### Changes proposed in this pull request: 
+
+ - 
+ - 
```

Comment:
Yes, unit tests! Code coverage of new code should show that most code paths are covered - if possible even error conditions.
I would also add that maybe the poster needs to squash all commits for the PR? I have mixed feelings there, sometimes it may help to split the work in several commits - even for review.